feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go

```diff
@@ -373,10 +381,19 @@ func TestTrafficWithGracefulRestartSpeaker(t *testing.T) {
 		t.Log("Start DUT interface Config")
 		configureDUT(t, dut)
 	})
+
+	t.Log("Configure Network Instance")
+	d := &telemetry.Device{}
+	ni := d.GetOrCreateNetworkInstance(*deviations.DefaultNetworkInstance)
+	ni.Type = telemetry.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE
+	ni.RouterId = ygot.String(dutDst.IPv4)
+	dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance).Type().Replace(t, ni.Type)
+	dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance).RouterId().Replace(t, *ni.RouterId)
+
 	// Configure BGP+Neighbors on the DUT
 	t.Run("configureBGP", func(t *testing.T) {
 		t.Log("Configure BGP with Graceful Restart option under Global Bgp")
-		dutConfPath := dut.Config().NetworkInstance("default").Protocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Bgp()
+		dutConfPath := dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance).Protocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Bgp()
 		fptest.LogYgot(t, "DUT BGP Config before", dutConfPath, dutConfPath.Get(t))
```

Comment:
Can you move this line, so that it is called after ```dutConfPath.Replace(t, dutConf)```.

This way we don't need to create a BGP instance before.

Reply:
Changed configure network instance code as below. please check,

t.Run("configureNetworkInstance", func(t *testing.T) {
		t.Log("Configure/update Network Instance")
		dutConfNIPath := dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance)
		dutConfNIPath.Type().Replace(t, telemetry.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)
		dutConfNIPath.RouterId().Replace(t, dutDst.IPv4)
	})